client/index.html

```diff
@@ -218,38 +218,20 @@ <h1 class="title">Settings</h1>
 							</div>
 							<div class="col-sm-6">
 								<label class="opt">
-									<input type="checkbox" name="join">
-									Show joins
+									<input type="checkbox" name="optional">
+									Show <abbr title="Joins, parts, kicks, nick changes, and mode changes">status messages</abbr>
```

Comment:
Then at the very least, we should refresh the page for this option.

At the moment, when enabling/disabling this option, one needs to realize it did not work, close the app, re-open the app (since there is no refresh button when app is added to homescreen on mobile).

Would you like me to implement that?